src/Cli/dotnet/commands/InstallingWorkloadCommand.cs

```diff
@@ -123,6 +123,13 @@ protected bool TryHandleWorkloadUpdateFromVersion(ITransactionContext context, D
                 _workloadInstaller.UpdateInstallMode(_sdkFeatureBand, true);
             }
 
+            // Delete the current advertising manifest because if we fail to find the right workload version, we want to fail.
+            var advertisingPackagePath = Path.Combine(_userProfileDir, "sdk-advertising", _sdkFeatureBand.ToString(), "microsoft.net.workloads");
```

Comment:
nit: Can you pull microsoft.net.workloads out into a variable with a clear name so we can tell this is the workload set folder?